ext/random/tests/03_randomizer/methods/getBytesFromString_fast_path.phpt

```diff
@@ -0,0 +1,112 @@
+--TEST--
+Random: Randomizer: getBytesFromString(): Fast Path Masking
+--FILE--
+<?php
+
+use Random\Engine\Test\TestWrapperEngine;
+use Random\Engine\Xoshiro256StarStar;
+use Random\Randomizer;
+
+require __DIR__ . "/../../engines.inc";
+
+$allBytes = implode('', array_map(
+    fn ($byte) => chr($byte),
+    range(0x00, 0xff)
+));
+
+// Xoshiro256** is the fastest engine available.
+$xoshiro = new Xoshiro256StarStar();
+
+var_dump(strlen($allBytes));
+echo PHP_EOL;
+
+// Fast path: Inputs less than or equal to 256.
+for ($i = 1; $i <= \strlen($allBytes); $i *= 2) {
```

Comment:
Is there a reason here that you use the FQN for the function and not above (same as line 37 and 60). This seems inconsistent.

Reply:
Good catch, no, there is not. My muscle memory code style has the leading backslash, but the tests do not usually use it. I'll remove it.